Index: src/main/java/org/example/controller/mapMenuController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/controller/mapMenuController.java b/src/main/java/org/example/controller/mapMenuController.java
deleted file mode 100644
--- a/src/main/java/org/example/controller/mapMenuController.java	(revision 1c8d357a60af2e4e6517bc1b3973eecc47202784)
+++ /dev/null	(revision 1c8d357a60af2e4e6517bc1b3973eecc47202784)
@@ -1,39 +0,0 @@
-package org.example.controller;
-
-public class mapMenuController {
-    private void setTexture(int x , int y,String type)
-    {
-
-    }
-    private void setTexture(int x1, int y1,int x2, int y2,String type)
-    {
-
-    }
-    private void clear()
-    {
-
-    }
-    private void dropRock()
-    {
-    }
-    private void dropUnit(int x , int y, String type)
-    {
-
-    }
-    private void dropBuilding(int x , int y)
-    {
-
-    }
-    private void dropTree(int x , int y, String type)
-    {
-
-    }
-    private void dropRock(int x , int y , char direction)
-    {
-
-    }
-    private boolean checkLand(int x , int y)
-    {
-        return true;
-    }
-}
Index: src/main/java/org/example/view/mapMenu.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/view/mapMenu.java b/src/main/java/org/example/view/mapMenu.java
deleted file mode 100644
--- a/src/main/java/org/example/view/mapMenu.java	(revision 1c8d357a60af2e4e6517bc1b3973eecc47202784)
+++ /dev/null	(revision 1c8d357a60af2e4e6517bc1b3973eecc47202784)
@@ -1,41 +0,0 @@
-package org.example.view;
-
-import java.util.regex.Matcher;
-
-public class mapMenu {
-    private void run(Matcher matcher)
-    {
-        //TODO
-        //
-        //TODO exit
-    }
-   /* private map showMap(int x,int y )
-    {
-
-        /*TODO map should be shown 10 up down left right
-        s : soldier
-        SG : small gatehouse
-        BG : big gatehouse
-        D : drawBridge
-        LT : lookoutTower
-        PT : perimeter tower
-
-/*
-
-
-
-        return null;
-    }
-    private map showDetails(int x ,int y)
-    {
-        // TODO : only a tile should be shown
-        return null;
-
-    }
-    // TODO : have a guide for map tiles
-
-    private void changeEnvironment(int x, int y)
-    {
-        //should check
-    }*/
-}
