Index: src/main/java/org/example/controller/GovernmentController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/controller/GovernmentController.java b/src/main/java/org/example/controller/GovernmentController.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/org/example/controller/GovernmentController.java	
@@ -0,0 +1,4 @@
+package org.example.controller;
+
+public class GovernmentController {
+}
Index: src/main/java/org/example/controller/mapMenuController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/controller/mapMenuController.java b/src/main/java/org/example/controller/MapMenuController.java
rename from src/main/java/org/example/controller/mapMenuController.java
rename to src/main/java/org/example/controller/MapMenuController.java
--- a/src/main/java/org/example/controller/mapMenuController.java	
+++ b/src/main/java/org/example/controller/MapMenuController.java	
@@ -2,8 +2,8 @@
 import org.example.model.gameData.map;
 import org.example.model.tile;
 
-public class mapMenuController {
-    public mapMenuController() {
+public class MapMenuController {
+    public MapMenuController() {
     }
 
     public void showMap(int x, int y) {
@@ -44,7 +44,9 @@
             System.out.println();
 
         }
-
+    }
+    private void setTexture(int x , int y,String type)
+    {
 
     }
 
Index: src/main/java/org/example/controller/mapChangesMenuController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/controller/mapChangesMenuController.java b/src/main/java/org/example/controller/MapChangesMenuController.java
rename from src/main/java/org/example/controller/mapChangesMenuController.java
rename to src/main/java/org/example/controller/MapChangesMenuController.java
--- a/src/main/java/org/example/controller/mapChangesMenuController.java	
+++ b/src/main/java/org/example/controller/MapChangesMenuController.java	
@@ -2,7 +2,7 @@
 import org.example.model.enums.LandType;
 import org.example.model.gameData.map;
 
-public class mapChangesMenuController {
+public class MapChangesMenuController {
     private void setTexture(int x , int y,String type)
     {
         for(LandType l : LandType.values())
Index: src/main/java/org/example/view/mapMenu.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/view/mapMenu.java b/src/main/java/org/example/view/MapMenu.java
rename from src/main/java/org/example/view/mapMenu.java
rename to src/main/java/org/example/view/MapMenu.java
--- a/src/main/java/org/example/view/mapMenu.java	
+++ b/src/main/java/org/example/view/MapMenu.java	
@@ -1,6 +1,7 @@
 package org.example.view;
 
 import org.example.InputScanner;
+<<<<<<< HEAD:src/main/java/org/example/view/mapMenu.java
 import org.example.Main;
 import org.example.controller.mapMenuController;
 import org.example.model.gameData.map;
@@ -23,16 +24,68 @@
                 showMap(Integer.parseInt(mapMenuMatcher.group("xCoordinate"))
                         , Integer.parseInt(mapMenuMatcher.group("yCoordinate")));
                     //TODO moving on map should be added
+=======
+
+import java.util.regex.Matcher;
+
+public class MapMenu {
+    private void run(Matcher matcher)
+    {
+        String command;
+        InputScanner scanner = new InputScanner();
+        command = scanner.getScanner().nextLine();
+        while (true)
+        {
+//            if(mapMenuEnum.getMatcher(command,mapMenuEnum.SHOW_MAP) != null)
+//                showMap(Integer.parseInt(matcher.group("xCoordinate"))
+//                        , Integer.parseInt(matcher.group("yCoordinate")));
+//
+
+>>>>>>> origin/main:src/main/java/org/example/view/MapMenu.java
 
 
         }
 
+<<<<<<< HEAD:src/main/java/org/example/view/mapMenu.java
     }
     private static void showMap(int x,int y)
+=======
+        //TODO
+        //
+        //TODO exit
+    }
+<<<<<<< HEAD:src/main/java/org/example/view/mapMenu.java
+    private static void showMap(int x,int y)
+=======
+
+    public void enterMapEnvironmentSettings() {
+
+    }
+   /* private map showMap(int x,int y )
+>>>>>>> origin/mahdi:src/main/java/org/example/view/MapMenu.java
+>>>>>>> origin/main:src/main/java/org/example/view/MapMenu.java
     {
         mapMenuController controller = new mapMenuController();
         controller.showMap(x , y);
 
+<<<<<<< HEAD:src/main/java/org/example/view/mapMenu.java
+=======
+    }
+
+        /*TODO map should be shown 10 up down left right
+        s : soldier
+        SG : small gatehouse
+        BG : big gatehouse
+        D : drawBridge
+        LT : lookoutTower
+        PT : perimeter tower
+
+/*
+
+
+
+        return null;
+>>>>>>> origin/main:src/main/java/org/example/view/MapMenu.java
     }
 
     private map showDetails(int x , int y)
Index: src/main/java/org/example/model/building/buildingName.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/model/building/buildingName.java b/src/main/java/org/example/model/building/BuildingName.java
rename from src/main/java/org/example/model/building/buildingName.java
rename to src/main/java/org/example/model/building/BuildingName.java
--- a/src/main/java/org/example/model/building/buildingName.java	
+++ b/src/main/java/org/example/model/building/BuildingName.java	
@@ -1,11 +1,8 @@
 package org.example.model.building;
 
-import org.example.model.enums.Source;
-import org.example.model.enums.products;
+import static org.example.model.enums.Products.*;
 
-import static org.example.model.enums.products.*;
-
-public enum buildingName {
+public enum BuildingName {
     //TODO the rate & capacity & cost should be specified for each building
     MILL(20, WOOD, 0, null, 1,  FLOUR, null , 100),
     IRON_MINE(20, WOOD, 0, null, 2, IRON, null, 100),
@@ -110,7 +107,7 @@
     }
 
     //productive building
-    buildingName(int numberOfMaterial1, products material1Name, int numberOfMaterial2, products material2Name,
+    BuildingName(int numberOfMaterial1, products material1Name, int numberOfMaterial2, products material2Name,
                  int numberOfWorkers, products good1, products good2, int rate) {
         this.numberOfMaterial1 = numberOfMaterial1;
         this.material1Name = material1Name;
@@ -122,7 +119,7 @@
         this.rate = rate;
     }
     //storage building
-    buildingName(int numberOfMaterial , products materialName , int capacity)
+    BuildingName(int numberOfMaterial , products materialName , int capacity)
     {
         this.numberOfMaterial = numberOfMaterial ;
         this.materialName = materialName ;
@@ -130,13 +127,13 @@
 
     }
     // war buildings
-    buildingName(int fireRange , int defendRange)
+    BuildingName(int fireRange , int defendRange)
     {
         this.fireRange =  fireRange;
         this.defendRange = defendRange;
     }
     //city buildings
-    buildingName(int popularityRate,int numberOfMaterial1, products material1Name , int numberOfMaterial2, products material2Name)
+    BuildingName(int popularityRate, int numberOfMaterial1, products material1Name , int numberOfMaterial2, products material2Name)
     {
         this.numberOfMaterial = numberOfMaterial1;
         this.materialName = material1Name ;
@@ -144,13 +141,13 @@
         this.numberOfMaterial2 = numberOfMaterial2 ;
         this.material2Name = material2Name ;
     }
-    buildingName(int damage , int numberOfMaterial , products materialName)
+    BuildingName(int damage , int numberOfMaterial , products materialName)
     {
         this.damage = damage;
         this.numberOfMaterial = numberOfMaterial ;
         this.materialName = materialName ;
     }
-    buildingName(int numberOfMaterial1 , products material1Name , int numberOfMaterial2 , products material2Name, int numberOfWorkers)
+    BuildingName(int numberOfMaterial1 , products material1Name , int numberOfMaterial2 , products material2Name, int numberOfWorkers)
     {
         this.numberOfMaterial1 = numberOfMaterial1 ;
         this.material1Name = material1Name ;
@@ -160,7 +157,7 @@
 
     }
     //educations
-    buildingName(int costForEachPerson , int costForEachPerson2 , int numberOfMaterial1 , products material1Name,int numberOfMaterial2, products material2Name)
+    BuildingName(int costForEachPerson , int costForEachPerson2 , int numberOfMaterial1 , products material1Name, int numberOfMaterial2, products material2Name)
     {
         this.costForEachPerson = costForEachPerson;
         this.costForEachPerson2 = costForEachPerson2 ;
Index: src/main/java/org/example/controller/dropBuildingController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/controller/dropBuildingController.java b/src/main/java/org/example/controller/DropBuildingController.java
rename from src/main/java/org/example/controller/dropBuildingController.java
rename to src/main/java/org/example/controller/DropBuildingController.java
--- a/src/main/java/org/example/controller/dropBuildingController.java	
+++ b/src/main/java/org/example/controller/DropBuildingController.java	
@@ -1,12 +1,11 @@
 package org.example.controller;
-import org.example.model.enums.*;
 import org.example.model.building.*;
 import org.example.model.gameData.*;
 
-public class dropBuildingController {
-    public void dropProductiveBuilding(int x , int y , buildingName name)
+public class DropBuildingController {
+    public void dropProductiveBuilding(int x , int y , BuildingName name)
     {
-        for(buildingName building : buildingName.values()) {
+        for(BuildingName building : BuildingName.values()) {
             if (building.equals(name)) {
                 Building newBuilding = new ProductiveBuilding(name.toString(), 100, x, y, building.getMaterial1Name()
                         , building.getMaterial2Name(), building.getNumberOfMaterial1(), building.getNumberOfMaterial2(),
Index: src/main/java/org/example/controller/GovernmentController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/controller/GovernmentController.java b/src/main/java/org/example/controller/GovernmentController.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/org/example/controller/GovernmentController.java	
@@ -0,0 +1,4 @@
+package org.example.controller;
+
+public class GovernmentController {
+}
